src/module/actor/actor-sheet.js

```diff
@@ -418,6 +418,8 @@ export default class OseActorSheet extends ActorSheet {
     } else {
       const itemData = createItem(type);
       if (treasure) itemData.system = { treasure: true };
+      // when creating a new spell on the character sheet, we need to set the level
+      if (type === "spell") itemData.system = lvl ? { lvl } : {lvl: 1};
```

Comment:
I'm surprised this isn't getting some kind of default, weird. I wonder if we could add the default to the data model? At the very least to take the burden of setting a default off of the sheet class.

Reply:
Yeah, from 0.3.x to V10 I believe it used the value provided in `template.json` as the default. In V11, it appears to ignore that value. 

Why? At some point I know I heard Atropos say "template.json is going to go away," and I think we're seeing an early effect of that here. I believe now that we're using data models, template.json might as well not exist? That may have been the only relevant bit left

Reply:
https://www.youtube.com/watch?v=fYFvDuLvXGk&t=4170s here

Reply:
Is there some due diligence I should be doing to push this through? Like looking for more places where defaults are ignored?

Reply:
For now let's treat other places where this could happen as a separate ticket. Thanks!